src/Controller/RequestController.php

```diff
@@ -91,4 +93,29 @@ private function issetPost(): array
         }
         return $myPost;
     }
+
+    public function takeCare()
+    {
+        $errors = [];
+
+        if ($_SERVER['REQUEST_METHOD'] === 'POST' && isset($_POST['userId']) && isset($_POST['requestId'])) {
+            if ($_POST['userId'] != '-- --') {
+                $answererId = trim($_POST['userId']);
+                $requestId = trim($_POST['requestId']);
+                $answererId = (int)$answererId;
+                $requestId = (int)$requestId;
```

Comment:
You should check if they are integers instead of blindly casting them.
May be using filter_input and the FILTER_VALIDATE_INT ?